apps/arweave/src/ar.erl

```diff
@@ -328,16 +328,16 @@ parse_cli_args(["local_peer", Peer | Rest], C = #config{ local_peers = Peers })
 parse_cli_args(["sync_from_local_peers_only" | Rest], C) ->
 	parse_cli_args(Rest, C#config{ sync_from_local_peers_only = true });
 parse_cli_args(["transaction_blacklist", File | Rest],
-	C = #config{ transaction_blacklist_files = Files } ) ->
+	C = #config{transaction_blacklist_files = Files}) ->
 	parse_cli_args(Rest, C#config{ transaction_blacklist_files = [File | Files] });
 parse_cli_args(["transaction_blacklist_url", URL | Rest],
-		C = #config{ transaction_blacklist_urls = URLs} ) ->
+		C = #config{transaction_blacklist_urls = URLs}) ->
 	parse_cli_args(Rest, C#config{ transaction_blacklist_urls = [URL | URLs] });
 parse_cli_args(["transaction_whitelist", File | Rest],
-		C = #config{ transaction_whitelist_files = Files } ) ->
+		C = #config{transaction_whitelist_files = Files}) ->
 	parse_cli_args(Rest, C#config{ transaction_whitelist_files = [File | Files] });
 parse_cli_args(["transaction_whitelist_url", URL | Rest],
-		C = #config{ transaction_whitelist_urls = URLs} ) ->
+		C = #config{transaction_whitelist_urls = URLs}) ->
```

Comment:
Why do we remove spaces here, but not [here](https://github.com/ArweaveTeam/arweave/pull/502/files#diff-f148c1ce17ebe7e20d6eed136cc4f0c0e517b978a199351bb7e50aca826a8de4R452)

Reply:
that is a good question, I believe the complaint was just about the parentheses and I did more than was needed to satisfy this lint rule